fluent/backends/request/client.js

```diff
@@ -0,0 +1,226 @@
+'use strict'
```

Comment:
Is the plan to move all fluent development to this repo? If not, then let's eliminate any other backends...

If we are going to move development here, let's figure out how best to package this stuff.